homework2/task5.py

```diff
@@ -0,0 +1,15 @@
+# Реализуйте алгоритм перемешивания списка.
+
+from random import randint
+
+N = 20
+numbers = []
+for i in range(N):
+    numbers.append(randint(-N, N))
+print(numbers)
+for i in range(len(numbers)):
+    index = randint(0, len(numbers) - 1)
+    temp = numbers[i]
+    numbers[i] = numbers[index]
+    numbers[index] = temp
```

Comment:
```suggestion
    numbers[i], numbers[index] = numbers[index], numbers[i]
```
Это же Python. Пользуйтесь его преимуществами.